test/functional/rpc_blockchain.py

```diff
@@ -448,6 +448,11 @@ def _test_getnetworkhashps(self):
             """).strip(),
             lambda: self.nodes[0].getnetworkhashps("a", []),
         )
+        assert_raises_rpc_error(
+            -8,
+            textwrap.dedent("Block does not exist at specified height").strip(),
+            lambda: self.nodes[0].getnetworkhashps(100, 99999999),
```

Comment:
instead of using the hardcoded `99999999` we can use `self.nodes[0].getblockcount() + 1` which might be better

Reply:
Good point; implemented.